From 5745a3efb97a0c29d6b17e7bfa52c73adf29217f Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Tue, 2 Apr 2019 13:55:18 +0200
Subject: [PATCH 133/386] arm64: dts: cn9131: disable SFP for CP1 eth0

Recently enabled CONFIG_SFP unveiled a problem with CP1 eth0 SFP cage
connectivity - I2C lanes are not connected (R192/R193 not mounted
on the PCB). Because of that the port became unusable. Apply work-around
re-enabling normal functionality by temporary disabling the SFP node
in device tree.

Change-Id: I6257311f36d05171a27d9e9e294b6e0b94624cb0
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/6805
Tested-by: sa_ip-sw-jenkins
Tested-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9131-db.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
index f8f5cd04b79d..9bb5abd29b24 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
@@ -57,6 +57,7 @@
 		tx-fault-gpio = <&cp1_gpio1 8 GPIO_ACTIVE_HIGH>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&cp1_sfp_pins>;
+		status = "disabled";
 	};
 };
 
-- 
2.17.1

